src/main/java/com/teragrep/rlp_03/delegate/event/RelpEvent.java

```diff
@@ -56,7 +57,14 @@ public abstract class RelpEvent implements Consumer<FrameContext>, AutoCloseable
 
     protected RelpFrameTX createResponse(RelpFrame rxFrame, String command, String response) {
         RelpFrameTX txFrame = new RelpFrameTX(command, response.getBytes(StandardCharsets.UTF_8));
-        txFrame.setTransactionNumber(rxFrame.txn().toInt());
+
+        // TODO implement separate type for hint frames
```

Comment:
TODO: Create ticket if important, otherwise leave the note for future development